components/moderation/ListProfiles.tsx

```diff
@@ -142,54 +149,68 @@ export function InnerListProfiles({
     refresh()
   }
 
+  async function handleReject(record: any) {
+    setIsJustUpdated(record.id)
+    await downgradeOrganization(record.id)
+    clearUpdating()
+    filterValues["role"] === "pendingUpgrade" && setFilters({}, [])
+    refresh()
+  }
+
   return (
-    <List actions={<UserRoleToolBar />}>
-      <Datagrid
-        rowStyle={record => ({
-          backgroundColor:
-            record.role === "pendingUpgrade"
-              ? highlightPending
-              : record.id === getJustUpdated()
-              ? highlightRecent
-              : "",
-          animationName: record.id === getJustUpdated() ? "fadeOut" : "",
-          animationDelay: "2s",
-          animationDuration: "2s",
-          animationFillMode: "forwards"
-        })}
-      >
-        <TextField source="fullName" label="Organization" />
-        <TextField source="contact.email" label="email" />
-        <TextField source="contact.phoneNumber" label="phone" />
-
-        <TextField source="contact.website" label="website" />
-        <WithRecord
-          render={(record: Profile & RaRecord) => {
-            return (
-              <>
-                <div>{record.about}</div>
-                <Internal href={`/profile?id=${record.id}`}>
-                  view profile
-                </Internal>
-              </>
-            )
-          }}
-        />
-        <TextField source="role" />
-        <WithRecord
-          render={record => {
-            return record.role === "pendingUpgrade" ? (
+    <Datagrid
+      bulkActionButtons={false}
+      rowStyle={record => ({
+        backgroundColor:
+          record.role === "pendingUpgrade"
+            ? highlightPending
+            : record.id === getJustUpdated()
+            ? highlightRecent
+            : "",
```

Comment:
Quite a nit but I would recommend pulling this out into a boolean variable with a name... because I'm having a wee bit trouble following the nested ternary :)